Dockerfile

```diff
@@ -1,15 +1,37 @@
-FROM centos:latest
-MAINTAINER Henrik Feldt <henrik@haf.se>
+FROM ubuntu:latest
+MAINTAINER Sebastian Fialka <sebastian.fialka@sebfia.net>
 
-ENV MONO_VERSION 4.0.4.1
+ENV MONO_VERSION 4.2.1.102
 ENV FSHARP_VERSION 4.0.0.3
+ENV LIBUV_PREFIX /opt/libuv
+ENV LIBUV_VERSION v1.7.5
+ENV LIBUV_BASENAME libuv-$LIBUV_VERSION
+ENV LIBUV_ARCHIVE $LIBUV_BASENAME.tar.gz
+ENV LIBUV_ARCHIVE_URL dist.libuv.org/dist/$LIBUV_VERSION/$LIBUV_ARCHIVE
 
-RUN set -x && \
-    yum clean all && \
-    yum install -y yum-utils && \
-    rpm --import "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF" && \
-    yum-config-manager --add-repo http://download.mono-project.com/repo/centos/ && \
-    yum install -y mono-complete-$MONO_VERSION fsharp-$FSHARP_VERSION
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF && \
+    echo "deb http://download.mono-project.com/repo/debian wheezy main" | sudo tee /etc/apt/sources.list.d/mono-xamarin.list && \
+    echo "deb http://download.mono-project.com/repo/debian alpha main" | sudo tee /etc/apt/sources.list.d/mono-xamarin-alpha.list && \
```

Comment:
Now that mono 4.2.1.102 is out do we still need the `alpha` ?
https://github.com/mono/docker/blob/master/4.2.1.102/Dockerfile